### ISSUE TITLE
[BUG] Keyboard and Peripherals become non-responsive

### ISSUE THREAD

**Issue by dmajestic80** [flybywiresim/a32nx#1420]
**Mod Version**
 Latest stable version from the downloader

built: 2020-10-15T01:38:50+00:00
ref: refs/heads/master
sha: a72a977f0b5c74eab9396bab64cb16f6cc4c8120
actor: Benjozork
event_name: push

**Describe the bug**

This is really hard to describe but the issue has come about all of a sudden and never had this issue with any previous versions.
With build I posted above, if I have any liveries (liveries that I've always used) my keyboard and peripherals all become non-responsive. If i go into Controls page and check Sensitivity I can see all my controls are being recognised. If i calibrate using Windows Game controllers, everything is functioning as normal.

However once back in the flightdeck everything is dead, including the mouse being able to pan around. I can however use the curser to press buttons in the cockpit and I can click the top menu bar. My hat switches become dead, honeycomb yoke, TCA sidestick, rudders and throttles are all non-responsive. However the ESC key is the only key that works on the keyboard.

If I then go to the main menu, select another flight and use the 172 for example, the issue persists. 

So I deleted the mod and liveries from the community folder. Then added a previous mod I had installed with build details:

built: 2020-10-14T04:45:18+00:00
ref: refs/heads/master
sha: 5ac2b03ea8132e7f549c3210e33437feef91f453
actor: Benjozork
event_name: push

I then added back my liveries and it works as normal. But now I'm using an older build and can't update to anything newer.

I've tried both the dev and stable versions downloaded from the installer and the issue is with both.

**To Reproduce**
 

1. Most of this is explained baove
2.
3.

**Expected behavior**

**Actual behavior**

**References**

**Additional context**

Was this working before/when did the issue start occurring?

Is this a problem in the vanilla unmodded game?

Not understanding what was going on I decided to re-install MSFS. The default, using liveries works fine. As soon as I add in the latest mod version I get the issue.

Discord username (if different from GitHub):

**Comment by Benjozork** [comment]
Can you retry updating, I reverted a change that broke some stuff. Might be related.

**Comment by dmajestic80** [comment]
> 
> 
> Can you retry updating, I reverted a change that broke some stuff. Might be related.

Thanks mate. Trying now with:

built: 2020-10-17T02:56:44+00:00
ref: refs/heads/master
sha: 8675df0eedca8320e8f22ed1aa50bb4ba8db2a60
actor: Benjozork
event_name: push

Will advise

**Comment by dmajestic80** [comment]
> 
> 
> Can you retry updating, I reverted a change that broke some stuff. Might be related.

**TEST 1 - Latest build with no liveries**

So I deleted the old mod build and all liveries from the community folder and updated to the latest build from this link: https://github.com/flybywiresim/a32nx/releases/download/vmaster/A32NX-master.zip
built: 2020-10-17T05:47:40+00:00
ref: refs/heads/master
sha: eac80e5bc12ea67936ea5aa93c53256baff461a0
actor: Benjozork
event_name: push

I started at the gate from Cold&Dark. 

**Outcome:** 

Everything looks to be working and functioning normally. Keyboard and Peripheral issue is no longer present.

**TEST 2 - With liveries added**

Same test as above, except this time I have added a previously working livery (Qantas in my case)

**Outcome**

Once again, everything looks to be working and functioning normally. Keyboard and Peripheral issue is no longer present.

Thank you so much for your help :)
